internal/cache/metadata/stat_cache_test.go

```diff
@@ -158,12 +158,12 @@ func (t *StatCacheTest) FillUpToCapacity() {
 	m1 := &gcs.MinObject{Name: "taco"}
 	m2 := &gcs.MinObject{Name: "quesadilla"}
 
-	t.cache.Insert(m0, expiration)                    // size = 1394 bytes
-	t.cache.Insert(m1, expiration)                    // size = 1382 bytes (cumulative = 2776 bytes)
-	t.cache.AddNegativeEntry("enchilada", expiration) // size = 178 bytes (cumulative = 2954 bytes)
-	t.cache.Insert(m2, expiration)                    // size = 1406 bytes (cumulative = 4360 bytes)
-	t.cache.AddNegativeEntry("fajita", expiration)    // size = 172 bytes (cumulative = 4532 bytes)
-	t.cache.AddNegativeEntry("salsa", expiration)     // size = 170 bytes (cumulative = 4702 bytes)
+	t.cache.Insert(m0, expiration)                    // size = 1410 bytes
```

Comment:
are you sure that this is 1410 and not 1418 as should be with additional 24 bytes added per positive entry?

Reply:
min object has only name, so CRC32 pointer is nil.

Reply:
ok.